smarts/core/road_map.py

```diff
@@ -61,6 +61,11 @@ def bounding_box(self) -> Optional[BoundingBox]:
         """
         raise NotImplementedError()
 
+    @property
+    def has_overpasses(self) -> bool:
+        """Whether the map has lanes with overlapping z-coordinates."""
+        return False
```

Comment:
I think we should report this value in the waymo browser tool (when a scenario is "explored").

Reply:
I think this might be a bit misleading terminology-wise in the case of tunnels. (although perhaps any 3d dimension is an overpass technically)